src/enums.h

```diff
@@ -288,6 +288,10 @@ enum ConditionParam_t {
 	CONDITION_PARAM_SPECIALSKILL_MANALEECHAMOUNT = 53,
 	CONDITION_PARAM_AGGRESSIVE = 54,
 	CONDITION_PARAM_DRUNKENNESS = 55,
+	CONDITION_PARAM_CUSTOMSKILL_COOLDOWNREDUCTION = 56,
+	CONDITION_PARAM_CUSTOMSKILL_INCREASEDAMAGE = 57,
+	CONDITION_PARAM_CUSTOMSKILL_INCREASEHEALING = 58,
+	CONDITION_PARAM_CUSTOMSKILL_INCREADEMANARESTORING = 59,
```

Comment:
typo in `INCREADEMANARESTORING`.

Other than that, the naming is inconsistent.
Here are consistent examples:
* REDUCEDCOOLDOWN
* INCREASEDDAMAGE
* INCREASEDHEALING
* INCREASEDMANAGAIN

(verb in past tense + object)